txalaRecorder.scd

```diff
@@ -70,29 +153,31 @@ thisProcess.platform.recordingsDir = ~storage_path; // recordings will be stored
 	recChannels;
 };
 
+
+
 // 2. Onset detection synth
-SynthDef(\detector, { |in=0, threshold=0.15, relaxtime=0.05, blocksize=1024, gain=0|
-    var plank_lvl, plank_idx, freq, hasFreq, baton_lvl, onsets, delayedOnsets, fft;
-    var sdel = 0.04; // [threeinputs.scd] this is a small delay to skip the first chaotic miliseconds
+SynthDef(\detector, { |in=0, threshold=0.06, relaxtime=0.01, blocksize=1024, gain=0|
+	var plank_lvl, plank_idx, freq, hasFreq, baton_lvl, onsets, delayedOnsets, fft;
+	var sdel = 0.04; // [threeinputs.scd] this is a small delay to skip the first chaotic miliseconds
 	var plank_channels = ~getChannels.(\plk);
 
-    var baton = SoundIn.ar(in);
-    var planks = SoundIn.ar(plank_channels);
-    var planklevels = WAmp.kr(planks, sdel);
+	var baton = SoundIn.ar(in) * gain;
+	var planks = SoundIn.ar(plank_channels) * gain;
```

Comment:
Did you want to keep the `gain` in?

Reply:
I am not totally sure yet. I have to test properly, but I guess it does not harm to have it